src/lib.rs

```diff
@@ -539,6 +575,18 @@ impl error::Error for Error {
     }
 }
 
+// Passthrough Debug to Display, since errors should be user-visible
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
+        f.write_str(self.as_str())
+    }
+}
+
```

Comment:
This may help, I'm not sure. Can you explain why we need this? :)

Reply:
It does the exact same thing as before but will work even without `std` (where Error isn't present)